src/stacks-hierarchy/Root_ParkingViolationsReporting/bottom-sheets/SelectProviderBottomSheet.tsx

```diff
@@ -0,0 +1,110 @@
+import {ViolationsReportingProvider} from '@atb/api/types/mobility';
+import {BottomSheetContainer} from '@atb/components/bottom-sheet';
+import {Button} from '@atb/components/button';
+import {ScreenHeaderWithoutNavigation} from '@atb/components/screen-header';
+import {ThemeText} from '@atb/components/text';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {ScreenHeaderTexts, useTranslation} from '@atb/translations';
+import {ParkingViolationTexts} from '@atb/translations/screens/ParkingViolations';
+import {useState} from 'react';
+import {ScrollView, View} from 'react-native';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
+import {ProviderLogo} from '../components/ProviderLogo';
+import {SelectGroup} from '../components/SelectGroup';
+
+type Props = {
+  providers: ViolationsReportingProvider[];
+  close: () => void;
+  onSelect: (provider: ViolationsReportingProvider) => void;
+  qrScanFailed: boolean | undefined;
+};
+
+export const SelectProviderBottomSheet = ({
+  providers,
+  close,
+  onSelect,
+  qrScanFailed,
+}: Props) => {
+  const {t} = useTranslation();
+  const {theme} = useTheme();
+  const styles = useStyles();
+  const [selectedProvider, setSelectedProvider] =
+    useState<ViolationsReportingProvider>();
+
+  return (
+    <BottomSheetContainer>
+      <ScreenHeaderWithoutNavigation
+        leftButton={{
+          type: 'close',
+          onPress: close,
+          text: t(ScreenHeaderTexts.headerButton.close.text),
+        }}
+        color="background_1"
+        setFocusOnLoad={false}
+        title={t(ParkingViolationTexts.selectProvider.title)}
+      />
+      <>
+        <ThemeText style={styles.content}>
+          {qrScanFailed
+            ? t(ParkingViolationTexts.selectProvider.qrFailedDescription)
+            : t(ParkingViolationTexts.selectProvider.description)}
+        </ThemeText>
+        <ScrollView style={styles.providerList}>
+          <SelectGroup
+            style={styles.content}
+            items={providers}
+            onSelect={(providers) => setSelectedProvider(providers[0])}
```

Comment:
Could consider handling onSelect in SelectGroup so that if only one item can be selected, then only one item will be received by onSelect. (`setSelectedProvider(providers[0])` is a bit unintuitive to read)